jjb/install-rhel7.yaml

```diff
@@ -31,9 +31,16 @@
     builders:
       - shell: |
             #!/bin/bash -ex
+            virtualenv ansible.venv && source ansible.venv/bin/activate
+            pip install --upgrade setuptools
+            pip install 'ansible>=2.1'
+
             export PYTHONUNBUFFERED=1 # Enable real-time output for Ansible
             ansible-playbook -i localhost, -c local \
                 "${WORKSPACE}/cinch/cinch/playbooks/install-rhel7.yml" \
                 -e delete_venv="${DELETE_VENV}" \
                 -e latest_tip="${LATEST_TIP}" \
                 -e beaker_kerberos="${BEAKER_KERBEROS}"
+
+            deactivate
+            rm -rf ansible.venv/
```

Comment:
Let's remove this 'rm' here, since we have the workspace-cleanup plugin in use which will handle this for us.